### DISCUSSION TITLE
sdf icons for iconColor

### DISCUSSION THREAD

**Discussion by saeedghasemi72** [rnmapbox/maps#2604] [Q&A]
anyone can help me for use sdf icons? I wanna dynamically change icon with iconColor
doc's url is broken [sdf icon](https://github.com/rnmapbox/maps/blob/v10.0.0-beta.61/help/troubleshooting/usingRecolorableImagesInMapboxMaps)

**Comment by saeedghasemi72** [comment]
@mfazekas can you help me?

**Reply by mfazekas** [reply]
We'll need to fix the doc links to point to mapbox pages: 
https://docs.mapbox.com/help/troubleshooting/using-recolorable-images-in-mapbox-maps/

**Comment by saeedghasemi72** [comment]
@mfazekas ok but how to we can use this feature in rnmapbox?  :D

**Reply by mglonnro** [reply]
Yes! Can we load local asset png files as sdf in <Mapbox.Images> somehow?
